packages/fluentui/react-bindings/src/FocusZone/FocusZone.tsx

```diff
@@ -128,7 +129,7 @@ export default class FocusZone extends React.Component<FocusZoneProps> implement
 
     // @ts-ignore
     this.windowElement = getWindow(this._root.current);
-    let parentElement = getParent(this._root.current);
+    let parentElement = getParent(this._root.current, ALLOW_VIRTUAL_ELEMENTS);
```

Comment:
any reason for this?

Reply:
v7's and v0's implementation of `getParent` is the same except that v0 doesn't allow virtual elements. v7's `getParent` has the second parameter, which allows virtual elements, set to true by default. In order to preserve v0's functionality as is we need to pass `false` to the second parameter, which is the value that is set to `ALLOW_VIRTUAL_ELEMENTS`.